inst/rmarkdown/templates/springer/skeleton/sn-apacite.bst

```diff
@@ -0,0 +1,5357 @@
+%%
```

Comment:
Is this file still us somewhere ?

Reply:
What do you mean?

Reply:
Sorry for the typo.  I meant

"Is this file still use somewhere ?"

Reply:
And now I understand that it is required by the classoption. All good !